sunpy/map/map_factory.py

```diff
@@ -172,6 +179,10 @@ def _read_file(self, fname, **kwargs):
                 data = filedata
                 meta = MetaDict(filemeta)
                 new_pairs.append((data, meta))
+
+        if not new_pairs:
+            raise NoMapsInFileError(f"Reading the file '{fname}' found no HDUs with NAXIS > 1, so no maps could be loaded.")
```

Comment:
```suggestion
            raise NoMapsInFileError(f"Found no HDUs in '{fname}' with NAXIS > 1, so no maps could be loaded.")
```
Although the message might change, I think it can be made shorter.